src/ocm/components/AddHosts/day2Wizard/Day2StaticIP.tsx

```diff
@@ -0,0 +1,134 @@
+import { Grid, TextContent, Text, TextVariants } from '@patternfly/react-core';
+import React from 'react';
+import {
+  ClusterWizardStep,
+  CpuArchitecture,
+  InfraEnv,
+  InfraEnvUpdateParams,
+  WizardFooter,
+} from '../../../../common';
+import useInfraEnv from '../../../hooks/useInfraEnv';
+import { InfraEnvsService } from '../../../services';
+import { FormViewHosts } from '../../clusterConfiguration/staticIp/components/FormViewHosts/FormViewHosts';
+import { FormViewNetworkWide } from '../../clusterConfiguration/staticIp/components/FormViewNetworkWide/FormViewNetworkWide';
+import {
+  StaticIpFormState,
+  StaticIpViewProps,
+} from '../../clusterConfiguration/staticIp/components/propTypes';
+import StaticIpViewRadioGroup from '../../clusterConfiguration/staticIp/components/StaticIpViewRadioGroup';
+import { YamlView } from '../../clusterConfiguration/staticIp/components/YamlView/YamlView';
+import { StaticIpInfo, StaticIpView } from '../../clusterConfiguration/staticIp/data/dataTypes';
+import { getStaticIpInfo } from '../../clusterConfiguration/staticIp/data/fromInfraEnv';
+import { useModalDialogsContext } from '../../hosts/ModalDialogsContext';
+import { useDay2WizardContext } from './Day2WizardContext';
+import { Day2WizardNav } from './Day2WizardNav';
+
+export const Day2StaticIP = () => {
+  const { day2DiscoveryImageDialog } = useModalDialogsContext();
+  const wizardContext = useDay2WizardContext();
+  const [infraEnv, setInfraEnv] = React.useState<InfraEnv>();
+  const [confirmOnChangeView, setConfirmOnChangeView] = React.useState<boolean>(false);
+  const [viewChanged, setViewChanged] = React.useState<boolean>(false);
+  const [initialStaticIpInfo, setInitialStaticIpInfo] = React.useState<StaticIpInfo>();
+  const [formState, setFormState] = React.useState<StaticIpFormState>();
+  const { data, close } = day2DiscoveryImageDialog;
+  const cluster = data.cluster;
+  const { updateInfraEnv } = useInfraEnv(cluster.id, CpuArchitecture.x86);
+
+  React.useEffect(() => {
+    const doItAsync = async () => {
+      const { data: infraEnv } = await InfraEnvsService.getInfraEnv(
```

Comment:
You also need `day1CPuArchitecture`. You can obtain it from the `cluster` object.